### ISSUE TITLE
Enumerations are not supported on Windows yet

### ISSUE THREAD

**Issue by ks74** [php/doc-en#2117]
From manual page: https://php.net/language.enumerations.overview

---

Although the docs state (PHP 8 >= 8.1.0)
my Windows Php 8.2 installation cannot parse it properly

**Comment by damianwadley** [comment]
Sure, they're supported: I'm using them right now in 8.1, but for the the fun of it,
```
u:\p\PHP\php-8.2.0-nts-Win32-vs16-x64>php --version
PHP 8.2.0 (cli) (built: Dec  6 2022 15:25:41) (NTS Visual C++ 2019 x64)
Copyright (c) The PHP Group
Zend Engine v4.2.0, Copyright (c) Zend Technologies

u:\p\PHP\php-8.2.0-nts-Win32-vs16-x64>type example.php
<?php
enum Suit
{
    case Hearts;
    case Diamonds;
    case Clubs;
    case Spades;
}
var_dump(Suit::Hearts);
?>
u:\p\PHP\php-8.2.0-nts-Win32-vs16-x64>php example.php
enum(Suit::Hearts)
```

Check which version of PHP you _actually_ are using.

**Comment by ks74** [comment]
Sorry, it works as expected. The reason it didn't work for me yesterday is I copied examples from the docs and they have a lot of nonbreaking spaces in them, and they cause parse errors. Problem is the parse errors do not prompt for the real reason, they're like 
unexpected token "=" 
or 
unexpected token "case" 

The reason the nbsps are copied is that I use Chrome extension AutoCopy which keeps them if it has the option "copy without formatting" checked. I unchecked the option and now if I copy an example nbsps are replaced with spaces.
If I copy with Ctrl-C nbsps are replaced too.
